src/js/footer/changeBannerPadding/index.js

```diff
@@ -1,21 +1,22 @@
 // Adding padding at the top when opening and closing a banner starts here
-  const bannerDisclaimer = document.querySelector('.banner_disclaimer');
-  const pageWrapper = document.querySelector('.page-wrapper');
-
+document.addEventListener('DOMContentLoaded', function() {
+  let bannerDisclaimer = document.querySelector('.banner_disclaimer');
```

Comment:
```suggestion
  const bannerDisclaimer = document.querySelector('.banner_disclaimer');
```
can we `const` here? it would help performance wise.